### PR TITLE
#17 Exemplar issue

### PR DESCRIPTION
**Pull request by Cleop**
#17 provides a screenshot of an EMF issue with well laid out user story and acceptance criteria

### CODE REVIEW

**Review by iteles — CHANGES_REQUESTED** [review]
@Cleop We discussed adding in the link to the issue as well here 🙏

**Comment by Cleop** [comment]
@iteles - I've just popped that link in now, ready for your review again 😊

**Review by iteles — APPROVED** [review]
Thanks!

**Review by nelsonic — APPROVED** [review]
@Cleop looks good. thanks for using EMF (open source) issue. 👍

File: readme.md [hunk 1]
```diff
@@ -84,6 +84,19 @@ of both mobile and desktop views.
...
 <img src="https://user-images.githubusercontent.com/16775804/34988677-617ac428-fab7-11e7-9f07-3c47a84d2ab5.png" width=150px />
 
+### Real world exemplar issue
+
+The following issue has a clearly defined user story and detailed acceptance
+criteria:
+
+![image](https://user-images.githubusercontent.com/16775804/35054346-3732eba8-fba4-11e7-8cce-4f239e45104b.png)
+
+Find the original issue at: https://github.com/emfoundation/ce100-app/issues/894
```
Comment (nelsonic) [inline readme.md:94]: 👍

**Pull request merged**